pkg/utils/utils.go

```diff
@@ -59,6 +59,15 @@ var (
 	PublicStorageProviderID = "7993447f-687f-490d-875c-ac95e89a62a4"
 	// PublicStorageSpaceID is the space id used by the sharestorageprovider
 	PublicStorageSpaceID = "7993447f-687f-490d-875c-ac95e89a62a4"
+	// VirtualSpaceOwner is the non-existing user which is needed for public share impersonation
+	VirtualSpaceOwner = userpb.User{
+		Id: &userpb.UserId{
+			OpaqueId: "dbdf733d-427f-4ac3-8093-2cba5c307e4c",
+			Type:     userpb.UserType_USER_TYPE_INVALID,
+		},
+		DisplayName: "Virtual Space Owner",
+		Username:    "virtualowner",
```

Comment:
```suggestion
		DisplayName: "No Owner",
		Username:    "nobody",
```
Hm, should we use the nobody/nogroup analogy from unix? https://refspecs.linuxbase.org/LSB_3.0.0/LSB-PDA/LSB-PDA/usernames.html